README.md

```diff
@@ -854,6 +855,139 @@ registration: {
+    features.registration: 'true'
+
+    ```
+- **clientId:** Client Id pre-registered with Okta for the OIDC authentication flow
+
+    ```javascript
+    clientId: 'GHtf9iJdr60A9IYrR0jw'
+    ```
+Optional configurations :
+
+- **parseSchema:** Callback used to mold the JSON schema that comes back from Okta API.
+
+    ```javascript
+    /**
+     The callback function is passed 3 arguments:
+     @params : schema, onSuccess, onFailure
+     1) @param {Object} schema: json schema returned from the API.
+     2) @param {Function} onSuccess: success callback.
+     3) @param {Function} onFailure: failure callback. Note: accepts a errorObject that can be used to show form level or field level errors.
+    **/
+    parseSchema: function (schema, onSuccess, onFailure) {
+      // This example will add an additional field to the registration form
+        schema.profileSchema.properties.address = {
+          'type': 'string',
+          'description': 'Street Address',
+          'default': 'Enter your street address',
+          'maxLength': 255
+        };
+        schema.profileSchema.fieldOrder.push('address');
+        onSuccess(schema);
+    }
+    onFailure callback accepts a errorObject that can be used to show form level or field level errors
+    ```
+ - **preSubmit :** Callback used primarily to modify the request parameters sent to Okta API.
+
+    ```javascript
+    /**
+     The callback function is passed 3 arguments:
+     @params : postData, onSuccess, onFailure
+     1) @param {Object} postData: form data that will be posted to the registration API.
+     2) @param {Function} onSuccess: success callback.
+     3) @param {Function} onFailure: failure callback. Note: accepts a errorObject that can be used to show form level or field level errors.
+    **/
+    preSubmit: function (postData, onSuccess, onFailure) {
+      // This example will add @companyname.com to the email if user fails to add it during registration
+      return postData.username.indexOf('@acme.com') > -1 ? username : username + '@acme.com';
+    }
+    ```
+ - **postSubmit :** Callback used to primarily get control and to modify the behavior post submission to registration API .
+
+    ```javascript
+    /**
+     The callback function is passed 3 arguments:
+     @params : response, onSuccess, onFailure
+     1) @param {Object} response: response returned from the API post registration.
+     2) @param {Function} onSuccess: success callback.
+     3) @param {Function} onFailure: failure callback. Note: accepts a errorObject that can be used to show form level or field level errors.
+    **/
+    postSubmit: function (response, onSuccess, onFailure) {
+      // This example will log the server response to the browser console
+      console.log(response);
+      onSuccess(response);
+    }
+    ```
+- **onFailure and ErrorObject :** The onFailure callback accepts an error object that can be used to show a form level vs field level error on the registration form
```

Comment:
nit: **onFailure** and **ErrorObject:**

Reply:
Add a period at the end of the sentence.

Reply:
Fixed